routes/api-v1.php

```diff
@@ -17,10 +17,13 @@
 
 Route::post('signup', [AuthController::class, 'signUp']);
 Route::post('login', [AuthController::class, 'login']);
-
+Route::post('send-otp', [AuthController::class, 'sendOtp']);
 Route::post('forget-password', [AuthController::class, 'forgetPassword']);
 Route::post('reset-password', [AuthController::class, 'resetPassword']);
 
 Route::group(['middleware' => 'auth:sanctum'], function () {
+    Route::post('verify-otp', [AuthController::class, 'verifyOtp']);
     Route::get('me', [UserController::class, 'me']);
+    Route::post('me', [UserController::class, 'update']);
```

Comment:
Can we name this method as `updateProfile`? This way, if in the future we need to implement the functionality to update a user, we can use this method for that purpose.